styles.css

```diff
@@ -524,7 +643,9 @@ footer {
 
   #myrecentwork {
     margin-top: 232rem;
-    size: 64rem;
+
+    /* size: 64rem; */
+
     line-height: 114rem;
     font-weight: 700;
     text-transform: capitalize;
```

Comment:
- [ ] It would be better if we update the live version of the app to have the latest working app. Also, we can improve the link by using `[placeholder](link)` this will properly format the link. The README could be improved by adding instructions about how to clone and run the app. Nice job using the template :+1: but we have to customize it to fix each project or specific project